docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md

```diff
@@ -28,6 +29,279 @@
 | [v1.7.1](Data_Dictionary_Release_Notes.md#release-with-api-v171) | March 16, 2017 |
 | [v1.3.1](Data_Dictionary_Release_Notes.md#release-with-api-v131) | September 7, 2016 |
 
+## v3.4.0
+
+* __GDC Product__: GDC Data Dictionary
+* __Release Date__: July 23, 2025
+
+### New Features and Changes
+
+* Altered `demographic` Entity
+	* Changes made to `required`
+	* New permissable value: `sex_at_birth`
+	* Removed permissable value: `gender`
+	* Changes made to `deprecated`
+	* New permissable value: `gender`
+	* Removed permissable value: `premature_at_birth`
+	* Removed permissable value: `weeks_gestation_at_birth`
+	* Changes made to `properties`
+		* New property: `sex_at_birth`
+		* Removed property: `premature_at_birth`
+		* Removed property: `weeks_gestation_at_birth`
+		* Changes made to `population_group`
+			* New permissable value: `Unknown`
+			* New permissable value: `Not Reported`
+* Altered `family_history` Entity
+	* New property: `deprecated`
+	* Changes made to `properties`
+		* New property: `relationship_sex_at_birth`
+* Altered `other_clinical_attribute` Entity
+	* Changes made to `deprecated`
+	* Removed permissable value: `pregnancy_count`
+	* Removed permissable value: `viral_hepatitis_serologies`
+	* Changes made to `properties`
+		* New property: `fertility_history`
+		* Removed property: `pregnancy_count`
+		* Removed property: `viral_hepatitis_serologies`
+		* Changes made to `aids_risk_factors`
+			* New permissable value: `Unknown`
+			* New permissable value: `Not Reported`
+		* Changes made to `comorbidities`
+			* Changes made to `items`
```

Comment:
Script: we can remove this line and list level (as in, everything below gets shifted to the left).